packages/web-react/src/hooks/useSpacingStyle.ts

```diff
@@ -1,17 +1,21 @@
 import { SpacingProp, SpacingCSSProperties } from '../types';
 
-export function useSpacingStyle(spacing: SpacingProp | undefined, prefix: string): SpacingCSSProperties {
+export function useSpacingStyle(
+  spacing: SpacingProp | undefined,
+  prefix: string,
+  direction: undefined | 'x' | 'y' = undefined,
```

Comment:
We can define a type for this that can be converted to constants and used instead of these magical letters.